pkg/reconciler/taskrun/resources/taskresourceresolution.go

```diff
@@ -30,66 +26,16 @@ type ResolvedTaskResources struct {
 	TaskName string
 	Kind     v1beta1.TaskKind
 	TaskSpec *v1beta1.TaskSpec
-	// Inputs is a map from the name of the input required by the Task
-	// to the actual Resource to use for it
-	Inputs map[string]*resourcev1alpha1.PipelineResource
-	// Outputs is a map from the name of the output required by the Task
-	// to the actual Resource to use for it
-	Outputs map[string]*resourcev1alpha1.PipelineResource
 }
 
-// GetResource is a function used to retrieve PipelineResources.
-type GetResource func(string) (*resourcev1alpha1.PipelineResource, error)
-
 // ResolveTaskResources looks up PipelineResources referenced by inputs and outputs and returns
 // a structure that unites the resolved references and the Task Spec. If referenced PipelineResources
 // can't be found, an error is returned.
-func ResolveTaskResources(ts *v1beta1.TaskSpec, taskName string, kind v1beta1.TaskKind, inputs []v1beta1.TaskResourceBinding, outputs []v1beta1.TaskResourceBinding, gr GetResource) (*ResolvedTaskResources, error) {
+func ResolveTaskResources(ts *v1beta1.TaskSpec, taskName string, kind v1beta1.TaskKind) (*ResolvedTaskResources, error) {
```

Comment:
nice to have: this function is just returning its inputs in a specific struct. Can we simplify and get rid of it?

Reply:
By removing that, is it okay to leave `ResolvedTaskResources` in `taskresourceresolution.go`?

Reply:
minor but since there are no resources involved anymore, we could rename the func and the struct (and probably the file too)

Reply:
make sense, renamed to `taskresolution`